### PR TITLE
[250] Footer Container and better CSS formatting

### PR DESCRIPTION
**Pull request by W3stside**
Part of a waterfall - branched off #250 (terms and condition)

##### QA Testing
no testing right now

### CODE REVIEW

File: src/custom/pages/AppBody/index.tsx [hunk 1]
```diff
@@ -13,10 +12,5 @@ export const BodyWrapper = styled(BodyWrapperMod)`
 
 // The styled container element that wraps the content of most pages and the tabs.
 export default function AppBody({ children, className }: { children: React.ReactNode; className?: string }) {
-  return (
-    <BodyWrapper className={className}>
-      {children}
-      <Version />
-    </BodyWrapper>
-  )
+  return <BodyWrapper className={className}>{children}</BodyWrapper>
```
Comment (W3stside) [inline src/custom/pages/AppBody/index.tsx:15]: `Version` should never have been here - now that we have a `Footer` component it makes much more sense to home it there

File: src/custom/pages/App/AppMod.tsx [hunk 2]
```diff
@@ -31,10 +31,13 @@ import DarkModeQueryParamReader from 'theme'
 // import Vote from './Vote'
 // import VotePage from './Vote/VotePage'
 
+import Footer from 'components/Footer'
+
 const AppWrapper = styled.div`
   display: flex;
   flex-flow: column;
   align-items: flex-start;
+  min-height: 100vh;
```
Comment (W3stside) [inline src/custom/pages/App/AppMod.tsx:40]: this matches the `height` of the body wrapper to the vh of page and keeps our footer down without `position: fixed`-ing it. Position fixed is great until the content is in the way of any large height element, i.e Markdown, and makes everything feel bad man.

File: src/custom/components/Footer/index.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import styled from 'styled-components'
+
+import Version from '../Version'
+const FOOTER_URLS = [
+  { name: '© 2021 Gnosis' },
```
Comment (W3stside) [inline src/custom/components/Footer/index.tsx:7]: do we want this?

**Review by anxolin — COMMENTED** [review]
as we discussed, we might want to hold this one a bit since we probably will be able to get by just by using the menu.

**Comment by W3stside** [comment]
@anxolin what I can do is remove the footer links for now and leave the Versions content in

File: src/custom/components/Footer/index.tsx (hunk 3 above)
Comment (alfetopito) [reply src/custom/components/Footer/index.tsx:7]: And is it really copyright? (that's what the © means, right?)
I thought it was an opensource fork of an opensource project.

**Review by alfetopito — APPROVED** [review]

File: src/custom/components/Footer/index.tsx (hunk 3 above)
Comment (anxolin) [reply src/custom/components/Footer/index.tsx:7]: I asked anna, if we want "🐮 Powered by Gnosis Protocol"

**Review by anxolin — APPROVED** [review]
Approving, but yep, I would delete all the policies. And probably the copywrite too.

**Comment by W3stside** [comment]
@anxolin @alfetopito the footer is being removed but we can re-examine if we want some sort of footer comment about `Gnosis` in another pr

**Pull request merged**